src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -204,6 +217,24 @@ async def sink_buffer(sample: Sample) -> None:
         )
         self._resampler_task = asyncio.create_task(self._resampler.resample())
 
+    def time_bound_oldest(self) -> datetime:
+        """
+        Return the time bounds of the ring buffer.
+
+        Returns:
+            The timestamp of the oldest sample of the ring buffer.
+        """
+        return self._buffer.time_bound_oldest()
+
+    def time_bound_newest(self) -> datetime:
+        """
+        Return the time bounds of the ring buffer.
+
+        Returns:
+            The timestamp of the newest sample of the ring buffer.
+        """
+        return self._buffer.time_bound_newest()
```

Comment:
These are good candidates for properties too:

```suggestion
    @property
    def time_bound_oldest(self) -> datetime:
        """
        Return the time bounds of the ring buffer.

        Returns:
            The timestamp of the oldest sample of the ring buffer.
        """
        return self._buffer.time_bound_oldest()

    @property
    def time_bound_newest(self) -> datetime:
        """
        Return the time bounds of the ring buffer.

        Returns:
            The timestamp of the newest sample of the ring buffer.
        """
        return self._buffer.time_bound_newest()
```